CoreWiki/Migrations/20180419024248_InitialCreate.cs

```diff
@@ -4,21 +4,24 @@
 
 namespace CoreWiki.Migrations
 {
-    public partial class Initial : Migration
+    public partial class InitialCreate : Migration
```

Comment:
Let's not update the migration directly, but instead create a new migration that updates the existing database

Reply:
Yes, my bad. Being new to ef core & github I may of got a bit excited ;)